### PR TITLE
Do not update ansible venv all the time

### PR DESCRIPTION
**Pull request by toabctl**
When calling run.sh, currently the ansible venv in the workspace
directory is updated all the time.
This is time consuming and may not work when running multiple run.sh
steps in parallel.
So stop doing that and just use the available venv.

### CODE REVIEW

File: script_library/bootstrap-ansible-if-necessary.sh [hunk 1]
```diff
@@ -10,8 +10,11 @@ function install_ansible (){
     local socok8s_workspace=${SOCOK8S_WORKSPACE_BASEDIR}/${SOCOK8S_ENVNAME}-workspace
     if [[ ! -d ${socok8s_workspace}/.ansiblevenv/ ]]; then
         virtualenv ${socok8s_workspace}/.ansiblevenv/
+        source ${socok8s_workspace}/.ansiblevenv/bin/activate
```
Comment (jsuchome) [inline script_library/bootstrap-ansible-if-necessary.sh]: BTW, could you add explicit `pip install --upgrade pip`?

Comment (toabctl) [reply script_library/bootstrap-ansible-if-necessary.sh]: why?

Comment (jsuchome) [reply script_library/bootstrap-ansible-if-necessary.sh]: Well at least my pip is complaining about it and I have to do it manually

Comment (toabctl) [reply script_library/bootstrap-ansible-if-necessary.sh]: can we do that in another commit/PR ?

Comment (dirkmueller) [reply script_library/bootstrap-ansible-if-necessary.sh]: Yeah, in place upgrading pip is more fun.. let's not add it here

File: script_library/bootstrap-ansible-if-necessary.sh [hunk 2]
```diff
@@ -10,8 +10,11 @@ function install_ansible (){
...
+        pip install --upgrade -r $(dirname "$0")/script_library/requirements.txt
+        python -m ara.setup.env > ${socok8s_workspace}/.ansiblevenv/ara.rc
+    else
+        echo "Found virtualenv at ${socok8s_workspace}/.ansiblevenv . Using that"
```
Comment (dirkmueller) [inline script_library/bootstrap-ansible-if-necessary.sh]: NIT: 
```suggestion
        echo "Re-using existing virtualenv at ${socok8s_workspace}/.ansiblevenv"
```

**Pull request merged**